### PR TITLE
docs: add spike essential filesystem

### PR DESCRIPTION
**Pull request by cscovino**
There is a lot of changes because I changed the folder name of the spike prime to differentiate it from spike essential

### CODE REVIEW

**Comment by gpdaniels** [comment]
Thanks for your contribution!

I'm not sure about renaming all the `spike -` directories just in case that would break other links on the web.

Maybe the new filesystems could go in an `spike essential - ` set of directories?

File: README.md [hunk 1]
```diff
@@ -39,7 +39,6 @@ This applies to the SPIKE Essential small hub.
 |            **Gyroscope** | Three-axis gyroscope (IC: ?).                                                    |
 |                **Ports** | 2 LPF2 ports = 2 normal speed (115kB).                                           |
 
-
```
Comment (gpdaniels) [inline README.md:42]: Could you remove this change from the MR?

File: filesystem/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/etc/hostname [hunk 2]
```diff
@@ -0,0 +1 @@
+LEGO Hub@Esse
```
Comment (gpdaniels) [inline filesystem/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/etc/hostname:1]: Could you remove this file from the MR.

File: filesystem/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/projects/.slots [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (gpdaniels) [inline filesystem/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/projects/.slots:1]: Could you remove this file from the MR.

File: filesystem/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/runtime.log [hunk 4]
```diff
@@ -0,0 +1,2 @@
+{'i': 'edxs', 'm': 'remove_project', 'p': {'slotid': 0}}
+free memory: 69600 allocated memory: 182640
```
Comment (gpdaniels) [inline filesystem/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/runtime.log:2]: Could you remove this file from the MR.

File: firmware/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/modules.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+>>> help('modules')
+__main__          math              uhashlib          uselect
+_onewire          micropython       uheapq            ustruct
+_uasyncio         uarray            uio               usys
+builtins          ubinascii         ujson             utime
+cmath             ubluetooth        umachine          utimeq
+firmware          ucollections      uos               uzlib
+gc                uctypes           urandom
+hub               uerrno            ure
+Plus any modules on the filesystem
```
Comment (gpdaniels) [inline firmware/spike essential - v1.0.00.0070-51a2ff4 - 3.1.21/modules.txt:10]: I've not bothered including the `Plus any modules on the filesystem` line in ther other files.

**Review by gpdaniels — CHANGES_REQUESTED** [review]
Could you remove the couple of `.DS_Store ` files.

Also there's a number of firmware directory renames still in the MR.

**Comment by cscovino** [comment]
Ok. Sorry for that. I think it'd done. Let me know if there is any change to be made.

File: README.md (hunk 1 above)
Comment (gpdaniels) [reply README.md:42]: The blank line became a bullet point?

**Review by gpdaniels — CHANGES_REQUESTED** [review]
The firmware directories are still renamed. 

e.g.
```
firmware/spike - v0.5.01.0000-42a938e - 0.0.0/hub module.txt
```
to
```
firmware/spike prime - v0.5.01.0000-42a938e - 0.0.0/hub module.txt
```

File: README.md (hunk 1 above)
Comment (cscovino) [reply README.md]: Sorry, I deleted that bullet point.

**Comment by cscovino** [comment]
Sorry, I hope this time it's ok 😅

**Review by gpdaniels — APPROVED** [review]

**Comment by gpdaniels** [comment]
Thanks for your contribution! :)

**Pull request merged**